test/e2eTests/pages/registrationDashboardPage.js

```diff
@@ -1,9 +1,72 @@
 const Page = require('./page')
 
-module.exports = new Page({
-  isLoaded: {
-    element: '[ui-sref="dashboard.program"]',
-    urlPart: '/registration/#/dashboard',
-  },
-  components: ['patientSearch', 'checkIn'],
-});
+const LOG_TAG = '[RegistrationDashboardPage]';
+
+class RegistrationDashboardPage extends Page {
+
+  constructor() {
+    super({
+      isLoaded: {
+        element: '[ui-sref="dashboard.program"]',
+        urlPart: '/registration/#/dashboard',
+      },
+      components: ['patientSearch', 'checkIn'],
+    });
+
+    this.buttons = {
+      nextStep: { css: '#next-step' },
+      cancel: locate('button').find('span').withText('Cancel'),
+      addIdentifier: locate('button').find('span').withText(this.translate('REGISTRATION_ADD_IDENTIFIER')),
+      confirm: { css: '#confirm' },
+    };
+
+    this.tabs = {
+      identifiers: locate('a').withChild('span').withText('1'),
```

Comment:
Where is the locate function coming from?

Reply:
From CodeceptJS.
https://codecept.io/locators/

Reply:
nice! This will come in handy. Although this worries me a little bit because it makes our tests heavily dependent on the HTML structure.